tests/test_models.py

```diff
@@ -80,23 +93,25 @@ def test_create_csv(as_ops):
 # def test_filter_note_type():
 #     """Test filter_note_type function."""
 #     assert False
-#
-#
-# def test_replace_str():
-#     """Test replace_str function."""
-#     assert False
+
+
+def test_replace_str():
```

Comment:
If you keep this function (see comment above), you might consider adding a check that `old_string` is not in `new_value`, just to cover your testing bases.

Reply:
I'll do that if I keep it